third_party/2and3/typing_extensions.pyi

```diff
@@ -0,0 +1,32 @@
+import sys
+import typing
+from typing import (
+    ClassVar, ContextManager, Counter, DefaultDict, Deque,
```

Comment:
To re-export this name, you need to say `ClassVar as ClassVar` (this also goes for the rest of this file).